Update Pupil Capture docs to v2.0 (#377)
pupil-labs/pupil-docs: src/core/software/pupil-capture.md

```

## Pupil Detection

> Pupil's algorithms automatically detect the participant's pupil. It runs two detection pipelines in parallel, the 2D and the 3D pupil detection. The 2D detection uses common computer vision technology to detect the pupil location in the camera image. The 3D detection uses a 3D model of the eye(s) that constantly updates based on observations of the eye. This enables the system to compensate for movements of the headset - slippage. To build up an initial model, it is recommended to briefly look around your field of view when putting on the headset.

Reviewer: ```suggestion
Pupil Core's algorithms automatically detect the participant's pupil. It runs two detection pipelines in parallel, the 2D and the 3D pupil detection. 

2D detection uses computer vision technology to detect the pupil location in the camera image. 

3D detection uses a 3D model of the eye(s) that updates based on observations of the eye. This enables the system to compensate for movements of the Pupil Core eye tracking headset on the participant's face (also known slippage). To build up an initial model, we recommend to look around your field of view when putting on the headset.
```

* `Pupil Min/Max` : In `Algorithm Mode` the two red circles represent the min and max pupil size settings. The green circle visualizes the current apparent pupil size. Set the min and max values so the green circle (current pupil size) is within the min/max range for _all_ eye movements.
* `Intensity Range` : Defines the minimum "darkness" of a pixel to be considered as the pupil. The pixels considered for pupil detection are visualized in blue when in `Algorithm Mode`. Try to minimize the range so that the pupil is always fully covered while having as little leakage as possible outside of the pupil. Be aware that this is dependent on the brightness and therefore has a strong interaction with `Video Source/Sensor Settings/Absolute Exposure Time`.

::: tip
<v-icon large color="info">info_outline</v-icon>
Keep in mind that pupil size values are defined in pixels and are therefore dependent on the resolution settings of your sensor.
:::

## Calibration

### Calibration Process

> Pupil Core headsets have two types of cameras attached. One camera records the subject's field of vision - we call this the `world camera`. Additionally there are one or more cameras recording the participant's eye movements - we call this the `eye cameras`. The data collected during the calibration period is used afterwards to correlate the world camera with the eye cameras.

Reviewer: ```suggestion
Pupil Core headsets have two types of cameras attached. One camera records the subject's field of vision - we call this the `world camera`. Additionally there are one or more cameras recording the participant's eye movements - we call these the `eye cameras`. The data collected during the calibration period is used afterwards to correlate the world camera with the eye cameras.
```

    </div>
</div>

### Choreographies

> All calibrations work in the way that specific markers are presented to the participant while the participant is instructed to look at the markers as closely as possible. The way in which markers are presented is called `Choreography`. Pupil offers different choreographies for common use cases.

Reviewer: Not all calibrations are marker based. 

```suggestion
All calibrations require a participant to look at a specific point in the real world or on screen. The way in which markers are presented is called a `Choreography`. Pupil Core provides different choreographies for common use cases.
```

#### Screen Marker Calibration Choreography

> This is the default choreography, and a quick way to get started. It is best suited for close range eye-tracking in a narrow field of view.

Reviewer: ```suggestion
This is the default choreography, and a quick way to get started.
```

<video width="100%" controls class="mb-5">
  <source src="../../media/core/videos/clb-s.mp4" type="video/mp4">
</video>

1. Select the `Screen Marker` choreography
2. Select your `Monitor` (if more than 1 monitor)

> 3. Toggle `Use fullscreen` to use the entire extents of your monitor (recommended). You can adjust the scale of the pattern for a larger or smaller calibration target.

Reviewer: Is `Use fullscreen` the default?

Author: It is.
Maybe we remove this point and add it below?
See suggestions there.

4. Press `c` on your keyboard or click the blue circular `C` button in the left hand side of the world window to start calibration.
5. Follow the marker on the screen with your eyes. Try to keep your head still during calibration.
6. The calibration window will close when calibration is complete.

If your participants have trouble following the markers, you can adjust the `Marker size` or `Sample duration` parameters.

#### Calibration Marker

> If you're not going to do calibration on screen with the `Screen Marker` choreography, you will need to download the Pupil's Circular Calibration Marker. This marker can be automatically detected by Pupil software in the world scene video. 

Reviewer: ```suggestion
If you're not going to do calibration on screen with the `Screen Marker` choreography, you will need to download the Pupil's Circular Calibration Marker. This marker can be automatically detected by Pupil software in the world video. 
```

    </div>
</div>

### Choreographies

All calibrations require a participant to look at a specific point in the real world or on screen. The way in which markers are presented is called a `Choreography`. Pupil Core provides different choreographies for common use cases.

#### Screen Marker Calibration Choreography
This is the default choreography, and a quick way to get started.

<video width="100%" controls class="mb-5">
  <source src="../../media/core/videos/clb-s.mp4" type="video/mp4">
</video>

1. Select the `Screen Marker` choreography
2. Select your `Monitor` (if more than 1 monitor)
3. Toggle `Use fullscreen` to use the entire extents of your monitor (recommended). You can adjust the scale of the pattern for a larger or smaller calibration target.
4. Press `c` on your keyboard or click the blue circular `C` button in the left hand side of the world window to start calibration.
5. Follow the marker on the screen with your eyes. Try to keep your head still during calibration.
6. The calibration window will close when calibration is complete.

> If your participants have trouble following the markers, you can adjust the `Marker size` or `Sample duration` parameters.

Author: @willpatera as I suggested in the comment above:

```suggestion
If your participants have trouble following the markers, you can adjust the `Marker size` or `Sample duration` parameters. We recommend to enable `Use fullscreen` for best marker visibility.
```

<a :href="$withBase('/pdfs/v0.4_marker.pdf')" alt="Pupil Labs calibration marker v4.0" target="_blank" rel="noopener">Download Pupil Labs Calibration Marker v0.4</a> to print or display on smartphone/tablet screen.

#### Single Marker Calibration Choreography

> Calibrate using a single marker, either with a hand held physical markers or a digital marker on the screen. Gaze at the center of the marker and move your head in a spiral motion. You can also move your head in other patterns. This choreography enables you to quickly sample a wide range of gaze angles and cover a large range of your FOV.

Reviewer: ```suggestion
Calibrate using a single marker, either with a printed (physical) markers or a digital marker displayed on screen. Gaze at the center of the marker and move your head in a spiral motion. You can also move your head in other patterns. This choreography enables you to quickly sample a wide range of gaze angles and cover a large range of your FOV.
```

1. Select `Single Marker` choreography
2. Press `c` on your keyboard or click the blue circular `C` button on the left hand side of the world window to start calibration.
3. Look at the center of the marker.
4. Slowly move your head while gazing at the center of the marker. We have found that a spiral pattern is an efficient way to cover a large area of the FOV.
5. Press the `C` button on your keyboard  or show the stop marker to stop calibrating.

::: tip
<v-icon large color="info">info_outline</v-icon>

> If you're using a physical marker, make sure to select <code>Physical Marker</code> in the <code>Marker display mode</code>. Also make sure that you do <strong>not</strong> display the marker on the screen at the same time when using a printed marker. Two markers visible to the world view at the same time will result in an inaccurate calibration.

Reviewer: This is the default now, not sure we need this tip anymore.

Author: Agreed. I'll delete this.

:::

::: tip
<v-icon large color="info">info_outline</v-icon>
This paper introduces and evaluates this type of single marker calibration - <code>CalibMe: Fast and Unsupervised Eye Tracker Calibration for Gaze-Based Pervasive Human-Computer Interaction</code>
:::

#### Natural Features Calibration Choreography

> This choregraphy is for special situations and far distances. Usually not required.

Reviewer: ```suggestion
This choregraphy is used only in special situations.
```
